### PR TITLE
Tom - Experiment w npm

### LINKED ISSUE #27: Failure to return home from Confirmation Page
**Describe the bug**
Nav to Home on confirmation page is broken.

**To Reproduce**
1. Go to 'confirmation page'
2. Click on 'home.'
3. See error - it fails to return to home, returns a 404 error.

**Expected behavior**
It should return to the home page.

**Screenshots**

Screen Shot 2021-07-15 at 12 39 15 AM

**Desktop (please complete the following information):**
 - OS: [e.g. Mac Big Sur]
 - Browser [e.g. chrome

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by tom-SFSDI**
@Holman-StateFarm 
@ALang-c7dg   
please review. 

## The Bug Fix
- closes #27 by fixing path to home page on ```confirmation.html```

## Added some NPM Packages
- Updates to ```CONTRIBUTING.md``` to explain usage of eslint and live-server.
- Added live-server so that we can see live updates without reloading pages.
  - added script of ```npm run server``` to launch.
- Added eslint NPM package which necessitated a creation of 
```package-lock.json```
```.eslintrc.json```
```.eslintignore``` to ignore certain files
- changes to ```package.json``` to add scripts for ```npm run lint``` && ```npm run lint-fix```
 - Added eslint rules which generally stuck to Google style guide with exceptions listed below.
   - custom rules include
 
   ```"rules": {
        "indent": [
            "error",
            2
        ],
        "max-len": [
            "error",
            {
            "code": 80,
            "ignoreUrls": true,
            "ignoreComments": true,
            "ignoreTrailingComments": true,
            "ignoreStrings": true,
            "ignoreTemplateLiterals": true
        }
    ],
        "no-multiple-empty-lines": [
            "error", 
            {
                "max": 1,
                "maxEOF": 0
            }
        ],
        "no-trailing-spaces": "error",
        "quotes": [
            "error",
            "single",
            {
                "avoidEscape": true,
                "allowTemplateLiterals": true
            }
        ],
        "require-jsdoc": 0
    }

These rules essentially enforce the use of semicolons, enforces use of single quotes (or backticks) when creating strings, disallows trailing whitespace, allows for a single blank line, enforces a single blank line on end of file, & enforces max line length of 80 except under certain circumstances. **Note: ESLINT only applies to .js files**

### CODE REVIEW

File: src/script/confirmation.js [hunk 1]
```diff
@@ -2,37 +2,37 @@ const retrievedDestination = localStorage.getItem('savedDestination');
...
+  const convertedValue = dollarValue / btcValue;
+  return convertedValue;
 }
 
-function etherConvert(dollarValue) {
-  const etherValue = 2000
-  const convertEtherValue = dollarValue / etherValue
-  return convertEtherValue
-}
+// function etherConvert(dollarValue) {
```
Comment (Holman-StateFarm) [inline src/script/confirmation.js:34]: Did you comment this unused code out or did the eslint do that automatically?

Comment (tom-SFSDI) [reply src/script/confirmation.js:34]: I commented out. eslint has an error when any variable is created but then never used. 'no-unused-variables" or something like that.

**Review by ALang-c7dg — COMMENTED** [review]
looks like eslint added a lot semi colons for us.  Nice!

Let's talk through these changes this morning.

**Review by Holman-StateFarm — APPROVED** [review]
Adding the live server and eslint were great moves.  This will be cool to use going forward!

**Pull request merged**